### ISSUE TITLE
Add 'other' option to vehicle_type

### ISSUE THREAD

**Issue by schnuerle** [openmobilityfoundation/mobility-data-specification#518]
### Is your feature request related to a problem? Please describe.

Would like to align vehicle types between MDS and GBFS.  

Currently [GBFS v2.1rc has 5 types](https://github.com/MobilityData/gbfs/blob/master/gbfs.md#free_bike_statusjson), and [MDS 0.4.1 has 4 types](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/provider#vehicles).  

They are the same except GBFS has an `other` type.

### Describe the solution you'd like

Would like to add the `other` to vehicle_types.  It seems valuable so you don't have to make up your own type or pigeon hole into one of the 4, for example if you want to do use MDS with a new vehicle type like dockless cars, drones, etc.  Can be updated later away from `other` by providers/cities when the new vehicle type is added to the spec.

If possible, I'd like to include this in `1.0.0`.  Can be implemented in the Reconciliation PR #506.

### Is this a breaking change

 * No, non-breaking

### Impacted Spec

For which spec is this feature being requested?

 * `agency`
 * `policy`
 * `provider`

### Describe alternatives you've considered

Do nothing, leave the feature specified as-is.

**Comment by thekaveman** [comment]
Submitted a Draft PR for review against the Reconciliation work: https://github.com/lacuna-tech/mobility-data-specification/pull/14

**Comment by schnuerle** [comment]
MobilityData said on a call the intention is catch new vehicle types until they can be added to future releases.  

See line 270 in this PR change: https://github.com/NABSA/gbfs/pull/136/files#diff-b85f1f78eaba81b0af480ba6c878256eR270

Also note in this GBFS PR they are changing bike_id to vehicle_id to match MDS!

**Comment by schnuerle** [comment]
Conversation 6/18 Provider WG call: Agreed this was a good thing to add.

**Comment by thekaveman** [comment]
Once #506 is merged, I'll submit a new PR with the same changes as the linked PR, into the MDS repo.

**Comment by schnuerle** [comment]
Resolved and added with the #534 PR.
